tests/conftest.py

```diff
@@ -3903,6 +3907,7 @@ def finalizer():
             if config.REPORTING["collect_logs_on_success_run"]:
                 collect_ocs_logs("testcases", ocs=False, status_failure=False)
                 collect_ocs_logs("testcases", ocp=False, status_failure=False)
+                collect_pod_container_rpm_package("testcases")
```

Comment:
Would this run for all test cases? 
Maybe we could add an option to run it only for deployment tests?

Reply:
Made the changes